### PR TITLE
Create BSLet172.xml

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Created stub for BFBS 172

### CODE REVIEW

File: LondonBibleSociety/BSLet172.xml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            <p>Encoded according to TEI P5 Guidelines.</p>
+         </projectDesc>
+         <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="https://raw.githubusercontent.com/BetaMasaheft/Documentation/master/prefixDef.xml">
+            <xi:fallback>
+               <p>Definitions of prefixes used.</p>
+            </xi:fallback>
+         </xi:include>
+      </encodingDesc>
+      <profileDesc>
+         <langUsage><language ident="en">English</language></langUsage>
```
Comment (thea-m) [inline LondonBibleSociety/BSLet172.xml:83]: you use "gez" above, so it must be defined here:   `<language ident="gez">Gǝʿǝz</language>`

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! Two small remarks

**Review by thea-m — APPROVED** [review]

**Comment by thea-m** [comment]
The XML is fine, I won't merge these records until approved by a project member

**Comment by SophiaD-M** [comment]
again two different identifiers for ms - please change

**Comment by thea-m** [comment]
Thanks @SophiaD-M! Very crucial!

File: LondonBibleSociety/BSLet172.xml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BSLet169" type="mss">
```
Comment (SophiaD-M) [inline LondonBibleSociety/BSLet172.xml:4]: wrong identifier

**Review by SophiaD-M — CHANGES_REQUESTED** [review]

**Review by SophiaD-M — APPROVED** [review]

**Pull request merged**